README.md

```diff
@@ -203,4 +208,8 @@ void encryptAndRun() throws EvervaultException {
 
 * Added decryptionDomains config option
 
-* Deprecated `ignoreDomains` and `intercept` config options
\ No newline at end of file
+* Deprecated `ignoreDomains` and `intercept` config options
+
+### 3.1.0
```

Comment:
You'll need to update [build.gradle](https://github.com/evervault/evervault-java/blob/master/lib/build.gradle) as well. Also It looks like 3.1.0 already exists? I think it just wasn't added to the README

Reply:
Ah I didn't realise, thanks for catching!